swatch-core/src/main/java/org/candlepin/subscriptions/tally/UsageCalculation.java

```diff
@@ -41,66 +50,16 @@ public class UsageCalculation {
    *
    * <p>Note that already data is scoped to an account, so account is not included in the key.
```

Comment:
Looking for suggestions on how to make this javadoc less confusing since we now include `billingAccountId`